ludwig/schema/features/augmentation/utils.py

```diff
@@ -0,0 +1,157 @@
+from dataclasses import field
+
+from marshmallow import fields, ValidationError
+from marshmallow_dataclass import dataclass
+
+from ludwig.api_annotations import DeveloperAPI
+from ludwig.constants import TYPE
+from ludwig.schema import utils as schema_utils
+from ludwig.schema.features.augmentation.base import BaseAugmentationContainerConfig
+from ludwig.utils.registry import Registry
+
+
+@DeveloperAPI
+@dataclass(repr=False)
+class ImageAugmentationContainerConfig(BaseAugmentationContainerConfig):
```

Comment:
Also unnecessary atm, can be removed

Reply:
commented out, TODO to clean up before merge